packages/control-property-editor/src/i18n/i18n.json

```diff
@@ -73,13 +73,14 @@
   "SHOW_CHANGES": " the app preview to show those changes.",
   "CONFIRM_CHANGE_SUMMARY_DELETE_SUBTEXT": "Are you sure you want to delete the change for this property? This action cannot be undone.",
   "CONFIRM_OTHER_CHANGE_DELETE_SUBTEXT": "Are you sure you want to delete this {{name}} change? This action cannot be undone.",
-  "NO_CONTROL_CHANGES_FOUND": "No Control Changes Found",
+  "NO_CHANGES_FOUND": "No historic changes",
+  "MODIFY_CHANGES": "This application was not modified yet",
```

Comment:
Text keys should be a bit more descriptive and it seems a bit misleading as it seems it is related to making some changes not the description of the `NO_CHANGES_FOUND`. Since we have two related texts now instead perhaps we can have something like this `NO_CHANGES_FOUND_TITLE` and `NO_CHANGES_FOUND_DESCRIPTION`.